addons/test_mail/tests/test_performance.py

```diff
@@ -836,7 +836,7 @@ def test_message_format(self):
         messages.flush()
         messages.invalidate_cache()
 
-        with self.assertQueryCount(emp=16):
+        with self.assertQueryCount(emp=17):
```

Comment:
Why is adding the formatter increasing the query count? Isn't it supposed to be just a move?